cmd/lotus-bench/main.go

```diff
@@ -458,11 +475,13 @@ func main() {
 						fmt.Printf("unseal: %s  (%s)\n", bo.SealingResults[0].Unseal, bps(bo.SectorSize, bo.SealingResults[0].Unseal))
 					}
 				}
-				fmt.Printf("generate candidates: %s (%s)\n", bo.PostGenerateCandidates, bps(bo.SectorSize*abi.SectorSize(len(bo.SealingResults)), bo.PostGenerateCandidates))
-				fmt.Printf("compute epost proof (cold): %s\n", bo.PostEProofCold)
-				fmt.Printf("compute epost proof (hot): %s\n", bo.PostEProofHot)
-				fmt.Printf("verify epost proof (cold): %s\n", bo.VerifyEPostCold)
-				fmt.Printf("verify epost proof (hot): %s\n", bo.VerifyEPostHot)
+				if !c.Bool("skip-commit2") {
```

Comment:
so skip commit2 skips all the election post stuff?

Reply:
Yep (It currently requires ~160-180G of memory)